pkgs/tools/video/recyclarr/update.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl jq common-updater-scripts gnused nix coreutils
+
+set -euo pipefail
+
+latestVersion="$(curl -s "https://api.github.com/repos/recyclarr/recyclarr/releases?per_page=1" | jq -r ".[0].tag_name" | sed 's/^v//')"
```

Comment:
API calls should optionally take a GITHUB_TOKEN because they are easily rate limited.